### DISCUSSION TITLE
Dịch out-of-core

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#30] [Terminology] [answered]
Cụm này thường nằm trong `out-of-core learning`, `out-of-core algorithm`, và ám chỉ việc dữ liệu không nằm vừa trong RAM.
Vì cái tên này cũng có lý do lịch sử nên mình đề xuất giữ nguyên.

@mlbvn/handson-ml2

**Comment by minhduc0711** [comment]
giữ nguyên là `out-of-core` nhé mọi người

**Reply by ngcthuong** [reply]
Mình nghĩ có thể xét lại những trường hợp như này vì chương này mình mượn từ nhiều quá.

**Comment by minhduc0711** [comment]
@ngcthuong vậy dịch là `ngoài lõi` được không ạ?

**Reply by ngcthuong** [reply]
Ngoài bộ nhớ?

**Reply by ngcthuong** [reply]
ngoài bộ nhớ chính?

**Reply by cuongvng** [reply]
ngoài RAM?

**Reply by minhduc0711** [reply]
Quote wiki
> In computing, external memory algorithms or out-of-core algorithms are algorithms that are designed to process data that are too large to fit into a computer's main memory at once.

nên dịch là `ngoài bộ nhớ chính` cho generic cũng được.

**Comment by minhduc0711** [comment, marked as answer]
từ này được sửa thành `ngoài bộ nhớ chính`
